composer.json

```diff
@@ -25,7 +25,8 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^9.5",
-        "squizlabs/php_codesniffer": "^3.0"
+        "squizlabs/php_codesniffer": "^3.0",
+        "silverstripe/versioned": "^1.12"
```

Comment:
Adding this to to require-dev so that we can call ->publishRecursive() in unit tests